sympy/algebras/tests/test_quaternion.py

```diff
@@ -172,3 +172,14 @@ def test_quaternion_multiplication():
 
     assert z * q == z_quat * q
     assert q * z == q * z_quat
+
+
+def test_issue_16318():
+    #for rtruediv
+    q1 = Quaternion(1, 2, 3, 4)
+    q0 = Quaternion(0, 0, 0, 0)
+    raises(ValueError, lambda: q0.__rtruediv__(q1))
```

Comment:
This would never actually be called. Wouldn't `1/q0` call `__rtruediv__`? (I'm not familiar with the quaternion module, but I'm assuming real scalars are cast into real quaternions)

Reply:
Yes you're right. I thought that in order for this section of code to appear covered in codecov, i should directly call \_\_rtruediv\_\_. By making the change to 1/q0 tho and running a coverage test, i see that this is not the case. I made the changes and hope its ok now.
Thank you!